synapse/lib/hiveauth.py

```diff
@@ -242,6 +242,12 @@ async def setUserName(self, iden, name):
         user.name = name
         await user.node.set(name)
 
+        beheld = {
+            'iden': iden,
+            'valu': name,
+        }
+        await self.feedBeholder('user:name', beheld)
```

Comment:
should this be like `user:set` and have a `'type': 'name'` 
We have a few things that could be getting set on a user (like email).